app/room.py

```diff
@@ -0,0 +1,116 @@
+import random
+from os import sys, path
+
+from app.fellow import Fellow
+from app.staff import Staff
+from app.person import Person
+
+sys.path.append(path.dirname(path.dirname(
+    path.dirname(path.abspath(__file__)))))
+
+
+class Room():
+
+    total_number_of_rooms = 0
+    room_list = {}
```

Comment:
i have refactored it into a  list in order to uphold variable naming convention. Thanks for the feedback